tools/test/h5repack/testfiles/h5repack-help.txt

```diff
@@ -136,6 +136,7 @@ usage: h5repack [OPTIONS] file1 file2
         UD=<filter_number,filter_flag,cd_value_count,value1[,value2,...,valueN]>
             Required values: filter_number, filter_flag, cd_value_count, value1
             Optional values: value2 to valueN
+            filter_flag: 1 is OPTIONAL or 0 is MANDATORY
```

Comment:
I assume (or guess) that filter_flag 1 means filter is optional and 0 means filter is mandatory?  If so, maybe adding 'filter' would make the meaning clearer:

filter_flag:  1 filter is OPTIONAL or 0 filter is MANDATORY